public/tutorials/using-infura-custom-provider.md

```diff
@@ -1,136 +1,152 @@
-# Using Infura (or any custom provider)
+# Using Infura (or a custom provider)
 
-[Infura](https://infura.io/) is a hosted Ethereum node cluster that lets your users run your application without requiring them to set up their own Ethereum node or wallet. For security reasons Infura does not manage users' private keys, which means they cannot sign transactions on your behalf (or your users' behalf). Similarly, Truffle doesn't intuitively know how to sign your transactions -- it usually hands that task over to the Ethereum node -- so if you'd like to deploy your contracts using Infura, you must first tell Truffle how you'd like these transactions signed.
+[Infura](https://infura.io/) is a hosted Ethereum node cluster that lets your users run your application without requiring them to set up their own Ethereum node or wallet.
+![Infura](/tutorials/images/infura/infura.png)
 
-Fortunately libraries can help us along. In this example we'll use an [hd wallet mnemonic](https://en.bitcoin.it/wiki/Deterministic_wallet) to generate our list of wallets, and we'll use the [web3-provider-engine](https://github.com/MetaMask/provider-engine) to provide us with built-in signing functionality.
+You may not be familiar with Infura, but if you've used MetaMask you've used Infura, as it is the node that MetaMask connects to.
```

Comment:
`node` here is misused, as it generally refers to an individual computer. Infura hosts dozens of nodes. I'd say something like `but if you've used Metamask you've used Infura, as it is the Ethereum provider that powers Metamask` (or something like).